src/client/windshaft.js

```diff
@@ -261,7 +289,7 @@ export default class Windshaft {
     }
 
     _buildSelectClause (MNS) {
-        const columns = Object.keys(MNS).concat(['the_geom_webmercator', 'cartodb_id']);
+        const columns = Object.keys(MNS).concat(['the_geom_webmercator', ID_PROPERTY]);
```

Comment:
I love having `cartodb_id` as a constant ✨ we've defined it twice as `ID_PROPERTY` and `DEFAULT_ID_PROPERTY`. As a minor improvement I'd keep one of those, but it's just a suggestion, it's ok like this!

Reply:
Nice!, I'll keep just DEFAULT_ID_PROPERTY const from Metadata